### PR TITLE
Move Around Buttons On Title Screen

### PR DESCRIPTION
**Pull request by Lunarmagpie**
image

### CODE REVIEW

File: client/src/logic/messages.ts
Comment (Screaper91) [inline client/src/logic/messages.ts:1]: `MATCHMAKING_PRIVATE_GAME_JOIN` and `MATCHMAKING_PRIVATE_GAME_CREATE` are still in the `type Messages` array

File: client/src/logic/matchmaking/matchmaking-reducer.ts
Comment (Screaper91) [inline client/src/logic/matchmaking/matchmaking-reducer.ts:1]: the case `MATCHMAKING_PRIVATE_GAME_CREATE` needs to be removed

File: client/src/app/match-making/match-making.tsx [hunk 1]
```diff
@@ -35,6 +37,46 @@ function MatchMaking() {
...
+				returnText="Settings"
+				className={classNames(css.privateLobby)}
+			>
+				<div className={css.privateLobbyLeft}>
+					<p>Opponent Code</p>
+					<div className={css.code} onClick={handleCodeClick}>
+						{gameCode}
+					</div>
+					<p>Spectator Code</p>
+					<div className={css.code} onClick={handleCodeClick}>
```
Comment (Screaper91) [inline client/src/app/match-making/match-making.tsx:54]: This copies the opponent code, not the spectator code.

**Review by Screaper91 — CHANGES_REQUESTED** [review]
There are a couple build errors that need addressing. Otherwise my only complaints are the the right side of private lobby UI should be neater and the code `div`'s need the `<CopyIcon />` svg.

File: client/src/app/match-making/match-making.tsx (hunk 1 above)
Comment (Lunarmagpie) [reply client/src/app/match-making/match-making.tsx:54]: This has been fixed on master but needs to be merged.

**Comment by Lunarmagpie** [comment]
The UI isn't done but I fixed the build errors. It might be best to merge this in now since the UI changes are far enough along that there isn't serious issues.

**Review by Screaper91 — APPROVED** [review]
Sounds good to merge functionality now and fix the UI later

**Pull request merged**